src/types/item.resolvers.js

```diff
@@ -3,17 +3,27 @@ const dummyData = require('../dummyData');
 const q42 = dummyData().entities.Q42;
 
 module.exports = {
-  Query: {
-    item: (_, args, ctx) => {
+  Query: {  
+    item: () => {
       return q42;
     }
   },
   Item: {
-    label: (_, args, ctx ) => {
-      return q42.labels[ args.language ];
+    label: (_, args) => {
+      return q42.labels[args.language];
     },
-    claims: () => {
-      return [].concat(...Object.values(q42.claims));
+    claims: (_) => {
```

Comment:
Apparently the first argument a resolver takes is the result of the parent query. The [Apollo docs](https://www.apollographql.com/docs/graphql-tools/resolvers/#resolver-function-signature) call it `obj` but I think we need a better name than that :D

Reply:
Oh nice! I'm not sure `_` is the ideal name, but it'll do for now :D
I guess naming it whatever the parent node in the data model is called makes sense? "entity" for claims and label, "claim" or "statement" for references? :thinking:

Reply:
Yes that seems reasonable. I see in the docs it's sometimes referred to as `rootValue`.
I guess we can refactor it along the way.